bin/rails

```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/env ruby
+if ENV['RAILS_ENV'] == 'test'
```

Comment:
is this really necessary?

Reply:
You're right and it's really doesn't important